features/environment.py

```diff
@@ -0,0 +1,21 @@
+from subprocess import check_call
+
+
+def before_tag(context, tag):
+    if tag == 'cli':
+        commands = [
```

Comment:
IIRC, `tox` already implicitly takes care of building and installing the tool in the environment, unless it's configured not to do it. So I think, this won't be required.

Reply:
Thanks @csomh, I'll keep that in mind for my next push.